varia/sass/blocks/image/_style.scss

```diff
@@ -4,7 +4,7 @@
 	figcaption {
 		color: #{map-deep-get($config-global, "color", "foreground", "light")};
 		font-size: #{map-deep-get($config-global, "font", "size", "xs")};
-		line-height: #{map-deep-get($config-global, "font", "line-height", "xs")};
```

Comment:
I couldn't find the `line-height.xs` mapping anywhere, so I changed it to `--line-height-body`. Is it ok for this instance?

Reply:
Hmmm, that’s probably a typo. `--line-height-body` should be fine although, I think `--line-height-heading` might actually look nicer since it’s a caption and not regular paragraph.

Reply:
Done at 0a48e84e4ec57ba50ea3121259b77a780113bd77